tests/test_phreeqc.py

```diff
@@ -257,3 +257,29 @@ def test_equilibrate(s1, s2, s5_pH, s6_Ca, caplog):
     s6_Ca.equilibrate()
     assert s6_Ca.get_total_amount("Ca", "mol").magnitude != initial_Ca
     assert np.isclose(s6_Ca.charge_balance, 0, atol=1e-8)
+
+
+def test_equilibrate_ca_co3_liquid():
+    solution = Solution([["Ca+2", "0.01 mol/L"], ["CO3-2", "0.01 mol/L"]], volume="1 L", engine="phreeqc")
```

Comment:
When you create the `solution`, you are using an older (deprecated) syntax where the solutes are specified as nested lists. It still works, but I'd prefer to have newly added tests adopt the current and preferred syntax which is a `dict`, e.g.

`Solution({"Ca+2": "0.01 mol/L", "CO3-2": "0.01 mol/L")'

Can you change that for all the tests?